src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.RegularCompilationTracker_Generators.cs

```diff
@@ -419,6 +447,16 @@ static void CheckGeneratorDriver(GeneratorDriver generatorDriver, ProjectState p
 
                 Contract.ThrowIfFalse(additionalTexts.Length == projectState.AdditionalDocumentStates.Count);
             }
+
+            bool GeneratorFilter(GeneratorFilterContext context)
```

Comment:
```suggestion
            bool ShouldRunGeneratorFilter(GeneratorFilterContext context)
```

Lacking context it's a bit unclear what this is actually filtering.